wiki/osu!_Program_Files/User_configuration_file/fr.md

```diff
@@ -0,0 +1,230 @@
+
+***[Click to return to Main page](Osu!_Program_Files "wikilink")***
+
+<img src="CF_DEAD.jpg" title="fig:Combo Fire." alt="Combo Fire." width="200" /> Ceci est une liste d'option située dans le fichier osu!.<nomdevotrepc>.cfg. Beaucoup de ces options son disponibles en jeu et il est recommandé d'y modifier ces options par son biais. N'éditez ces valeurs manuellement qu'en cas de vrai besoin.
```

Comment:
I'd tell you to change the `<img>` tag to something else but idk what RaikireHiuduo is doing with the image right now... so just remove the `<img src="CF_DEAD.jpg" title="fig:Combo Fire." alt="Combo Fire." width="200" /> `